dashboard/test/controllers/api/v1/users_controller_test.rb

```diff
@@ -46,6 +46,49 @@ class Api::V1::UsersControllerTest < ActionController::TestCase
     assert_response 403
   end
 
+  test 'a post request to accept_data_transfer_agreement updates data transfer related user properties' do
+    sign_in(@user)
+    @user.data_transfer_agreement_accepted = false
+    refute @user.data_transfer_agreement_accepted
+    Timecop.freeze(DateTime.now) do
```

Comment:
Note: This is equivalent to just `Timecop.freeze do` without any arguments. ([docs](https://github.com/travisjeffery/timecop/blob/master/lib/timecop/timecop.rb#L28-L34))